spec/models/article_spec.rb

```diff
@@ -18,4 +18,19 @@
     expect(Article.first.desc_facet).to eq("desc_facet")
     expect(Article.first.geo_facet).to eq("geo_facet")
   end
+
+  it {should have_many(:tweets)}
+
+ xit "should build associated tweets from article url" do
+     article_data = [{
+      title: "title",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.